Date: Wed, 29 Oct 2025 14:33:30 +1300
Subject: [PATCH] flashcp: Don't define bool

GCC 15 complains:

  misc-utils/flashcp.c:46:13: error: 'bool' cannot be defined via 'typedef'
     46 | typedef int bool;
        |             ^~~~
  misc-utils/flashcp.c:46:13: note: 'bool' is a keyword with '-std=c23' onwards
  misc-utils/flashcp.c:46:1: warning: useless type name in empty declaration
     46 | typedef int bool;
        | ^~~~~~~

stdbool.h is already pulled in via common.h so just use that instead of
defining a bool type.
---
 misc-utils/flashcp.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/misc-utils/flashcp.c b/misc-utils/flashcp.c
index 3fddeb0..69f2a1d 100644
--- a/misc-utils/flashcp.c
+++ b/misc-utils/flashcp.c
@@ -43,9 +43,7 @@
 #include <mtd/mtd-user.h>
 #include <getopt.h>
 
-typedef int bool;
-#define true 1
-#define false 0
+#include "common.h"
 
 #define EXIT_FAILURE 1
 #define EXIT_SUCCESS 0
